src/rules/export.js

```diff
@@ -50,7 +50,13 @@ module.exports = function (context) {
         return
       }
       let any = false
-      remoteExports.forEach((v, name) => (any = true) && addNamed(name, node))
+      remoteExports.forEach((v, name) => (
+        (any = true) &&
```

Comment:
@jfmengels wouldn't quite work since `any` is used for the `No named exports found in module` check below.

This would work though:
```js
const anyNamed = remoteExports.length > 0;
const namedExports = remoteExports.filter((v, name) => name !== 'default');
namedExports.forEach((v, name) => addNamed(name, node));
```

Happy to change if you think it makes it more readable (I admit I was confused about the assignment in the condition)